beacon-chain/p2p/service.go

```diff
@@ -295,6 +295,17 @@ func (s *Service) Stop() error {
 	if s.dv5Listener != nil {
 		s.dv5Listener.Close()
 	}
+	// Save metadata to file if static peer id is enabled.
+	if s.cfg.StaticPeerID {
```

Comment:
was this in the spec that it needed to be saved this way, i feel like we don't need to save here, we only really need to save when we update the value